homeassistant/components/input_text/__init__.py

```diff
@@ -286,7 +286,7 @@ async def async_set_value(self, value):
         self._current_value = value
         self.async_write_ha_state()
 
-    async def async_update_config(self, config: dict) -> None:
+    async def async_update_config(self, config: ConfigType) -> None:
```

Comment:
Can `__init__` and `from_yaml` be adjusted to use `ConfigType`, to match in `input_text`?